packages/conform-dom/submission.ts

```diff
@@ -339,15 +339,15 @@ export type UpdateIntent<Schema = unknown> = {
 		  };
 };
 
-export type RemoveIntent<Schema extends Array<any> = any> = {
```

Comment:
These checks are intentional to make sure the user passed a field with an array type since they won't work with non-array field. We might be able to fix it by `Schema extends Array<any> | null | undefined` instead

Reply:
the suggestion you've proposed doesn't work btw.